tox.ini

```diff
@@ -263,8 +263,8 @@ envlist =
     ; below mean these dependencies are being used:
     ; 0: sqlalchemy>=1.1,<1.2
     ; 1: sqlalchemy~=1.4 aiosqlite
```

Comment:
Nit: Could you update comments to include what (2) version is being tested?

Reply:
Added in [bfc5c5e](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2976/commits/bfc5c5eb9f70e648998c2f160b50a205ee2608d6). Not sure why aiosqlite is in above line; didn't add it to the line for (2). Let me know if ok.